### PR TITLE
update go version in mapper Dockerfile

### PR DESCRIPTION
**Pull request by wbc6080**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
bump go version to 1.20 according to #5146 
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by Shelley-BaoYue** [comment]
Does the ubuntu version need to be updated as well? Other parts all use 18.04, although I'm not quite sure what the effect of using 16.04 is.  cc @fisherxu 
image

File: staging/src/github.com/kubeedge/mapper-framework/_template/mapper/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17-alpine AS builder
+FROM golang:1.20-alpine AS builder
```
Comment (fisherxu) [inline staging/src/github.com/kubeedge/mapper-framework/_template/mapper/Dockerfile:1]:
```suggestion
FROM golang:1.20.10-alpine3.18
```
Seems we used the above version in all other dockerfiles cc @Shelley-BaoYue

**Comment by fisherxu** [comment]
> Does the ubuntu version need to be updated as well? Other parts all use 18.04, although I'm not quite sure what the effect of using 16.04 is. cc @fisherxu image

+1, let's keep consistent with others

**Comment by wbc6080** [comment]
OK,I will fix them

**Review by fisherxu — COMMENTED** [review]
/approve
/assign @Shelley-BaoYue

**Pull request merged**